gradle.properties

```diff
@@ -0,0 +1,64 @@
+modName = Backpack
```

Comment:
You need to keep the name from Constants.gradle! Don't change the names because you think it sounds better.